synapse/lib/types.py

```diff
@@ -715,7 +715,7 @@ class PropValuType(DataType):
     def __init__(self, tlib, name, **info):
         DataType.__init__(self, tlib, name, **info)
         # TODO figure out what to do about tlib vs core issues
-        self._getPropNorm = getattr(tlib, 'getPropNorm', None)
```

Comment:
not sure i see why we're doing this?  if tlib isn't a DataModel, we'll just explode in norm() anyway ( due to reqPropNorm being a None )

Reply:
this design was inherited from XrefType (keeping style local to other code).  Its something that would be cleaned up once tlib/datamodel is joined together.

Reply:
based on discussion, while this is an antipattern, it will be going away